### PR TITLE
integrations: Add screenshots and notes for hellosign

### PR DESCRIPTION
**Pull request by anirudhjain75**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@anirudhjain75, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by trueskawka** [comment]
Hi @anirudhjain75!

Thanks for working on this!

Could you add a screenshot with the generated message, as described in the task?

Could you also read through your notes and correct typos? Thanks!

**Comment by anirudhjain75** [comment]
@trueskawka I have shown that its integrated
I can't do anything furthur because i have used all of free account

File: webhook-integrations/hellosign/notes.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+**Hellosign:**
+        * Hellosign is an e-signing tool for signing document virtually through internet instead of signing it by hand.
```
Comment (trueskawka) [inline webhook-integrations/hellosign/notes.md]: singing documents
instead of signing them

File: webhook-integrations/hellosign/notes.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        * Hellosign is not a free tool but a paid one though you get to sign 3 document for free.
```
Comment (trueskawka) [inline webhook-integrations/hellosign/notes.md]: you get to sign 3 documents

File: webhook-integrations/hellosign/notes.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        * Hellosign can be directly intergrated with slack and requestb.in.
```
Comment (trueskawka) [inline webhook-integrations/hellosign/notes.md]: Slack

File: webhook-integrations/hellosign/notes.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        * Below are some instruction to follow while integrating Hellosign with Slack and Requestb
```
Comment (trueskawka) [inline webhook-integrations/hellosign/notes.md]: instructions

Comment (trueskawka) [inline webhook-integrations/hellosign/notes.md]: Requestb.in

File: webhook-integrations/hellosign/notes.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+        :
+        -> First make an account on Hellosign.com .
+        -> Go to integrations option and click **connect** in Slack .
+        -> Configure API using url from requestb.in
```
Comment (trueskawka) [inline webhook-integrations/hellosign/notes.md]: Requestb.in

File: webhook-integrations/hellosign/notes.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+        -> Click on **'Sign Document'** which is on top right position.
```
Comment (trueskawka) [inline webhook-integrations/hellosign/notes.md]: in the top right

File: webhook-integrations/hellosign/notes.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+        -> Upload your document then enter the email address of target reciever and click on **send** after adding document title.
+        -> After reciever signing the document youll get a notification in slack and a **JSON** will be created in requestb.in
```
Comment (trueskawka) [inline webhook-integrations/hellosign/notes.md]: After the receiver sings the document you'll get a notification in Slack

Comment (trueskawka) [inline webhook-integrations/hellosign/notes.md]: Requestb.in

**Comment by trueskawka** [comment]
Hi @anirudhjain75!

Could you clean up the commit history for this PR? I know the task has been accepted, but the PR hasn't been merged yet.

**Comment by fazerlicourice7** [comment]
There is no fixture commit to go along with this. I can't right the code for the integration without that information. Also, the slack screen shot, doesn't show the report by HelloSign. If they aren't sending the report, as @anirudhjain75 claims, then what am I supposed to model the zulip message off of?

**Pull request closed without merging**